components/DetailCountry/index.js

```diff
@@ -30,8 +35,30 @@ export default function DetailCountry({
             >
               <path d="M9.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.2 288 416 288c17.7 0 32-14.3 32-32s-14.3-32-32-32l-306.7 0L214.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z" />
             </svg>
-          </StyledButton>
+          </StyledBackButton>
         </Link>
+        <CenterDiv>
+          <button onClick={newImg}>+</button>
+        </CenterDiv>
+        <dialog id="newImageDialog">
+          <p>Bilder hinzufügen: </p>
+          <form method="dialog" onSubmit={addImage}>
+            <StyledButtonDiv>
+              <CldUploadButton
+                name="addimg"
+                id="addimg"
```

Comment:
Maybe you should use other designations, because it looks as if you made a typing mistake (-> "adding")